client/sites-dashboard-v2/site-preview-pane/dotcom-preview-pane.tsx

```diff
@@ -33,7 +33,8 @@ const DotcomPreviewPane = ( {
 }: Props ) => {
 	const { __ } = useI18n();
 
-	const isDotcomSite = !! site.is_wpcom_atomic || !! site.is_wpcom_staging_site;
```

Comment:
`isDotcomSite` is confusing since it returns false for Simple sites, so I changed it to `isAtomicSite`.

Reply:
Great. Thanks